<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="lstInputNumbers"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="deviceLayout">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="lstOutputNumbers"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QSpinBox" name="spinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnStart">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnStop">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnNextTime">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Следующий такт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRandomPair">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Добавить случайную пару</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnGenNPairs">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Генерация N пар</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="title">
           <string>Режим</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
           <item>
            <widget class="QRadioButton" name="rbtnConveyorType">
             <property name="text">
              <string>конвейер</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbtnLineType">
             <property name="text">
              <string>последовательный</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnResetConveyor">
          <property name="text">
           <string>Сбросить конвейер</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>426</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
